### ISSUE TITLE
Index empty field values as empty rather than "No" 

### ISSUE THREAD

**Issue by sam327** [algolia/algoliasearch-magento#616]
I'm using algolia 1.7.0. 

When algolia inserts _dropdown_ fields into the algolia index, if the field is blank, it enters a value of "No" - Instead, I think it should index a blank or null. 

Here's the use case where I'm running into a problem. 

I've added a field to my search results to show the manufacturer. So my newer template code looks like:

```
<div class="result-thumbnail">
    {{#image_url}}<img src="{{{ image_url }}}"/>{{/image_url}}
    {{^image_url}}<span class="no-image"></span>{{/image_url}}
</div>
{{#_highlightResult.manufacturer}}
    <div class="manufacturer">{{{_highlightResult.manufacturer.value}}}</div>
{{/_highlightResult.manufacturer}}
<div class="name">{{{_highlightResult.name.value}}}</div>
```

About 95% of my results have manufacturers, so it's 95% fine. But for the handful of products, there's no manufacturer, and in this field I get a "No" result - where instead I get a blank line.

So I'm seeing results like:

_[image]
No
White Socks_

and

_[image]
Levi's
Blue Jeans_

It looks like there's no conditional other than if-exists / if-doesn't-exist option in the hogan.js/mustache.js

Suggestions?

**Comment by JanPetr** [comment]
Hello @sam327,
there is an option in Algolia configuration by which you can disable indexing of "No"/empty values:
<img width="696" alt="screenshot 2016-10-15 16 23 47" src="https://cloud.githubusercontent.com/assets/1058144/19410744/c6dc9db4-92f3-11e6-80e7-811da45b60ce.png">
But currently the implementation removes empty values only from products which are composed by other products.
In upcoming version the implementation should change to be able to remove empty values from all attributes and products.

In case it does not currently work for your Manufacturer attribute, you can edit the code to make it work for everything. Here's the current code which removes empty values:
https://github.com/algolia/algoliasearch-magento/blob/master/app/code/community/Algolia/Algoliasearch/Helper/Entity/Producthelper.php#L594

**Comment by JanPetr** [comment]
Fixed in [1.7.2](https://github.com/algolia/algoliasearch-magento/releases/tag/1.7.2)
